package.json

```diff
@@ -34,6 +35,9 @@
     "grunt": "^0.4.5",
     "grunt-contrib-jshint": "~0.9.0",
     "grunt-jsonlint": "1.0.4",
-    "jqUnit": "git+https://github.com/fluid-project/node-jqUnit.git"
+    "grunt-shell": "0.6.4",
+    "grunt-gpii": "git://github.com/GPII/grunt-gpii.git#58ae055833f437a9e91baedb9f39a22c5fdd727f",
+    "jqUnit": "git+https://github.com/fluid-project/node-jqUnit.git",
```

Comment:
I imagine this should be bound to a specific version too, like the other git dependencies